sql/updates/world/3.3.5/2016_05_12_00_world_PROCS.sql

```diff
@@ -0,0 +1,10 @@
+-- Blazing Speed
+DELETE FROM spell_ranks where first_spell_id = 31641;
+INSERT INTO spell_ranks VALUES ( 31641,31641,1),( 31641,31642,2);
```

Comment:
You don't need to insert spell_ranks data for talents - it's automatically generated during startup
Inserting data to spell_ranks table for talents causes starup errors 

> The spell %u (rank: %u, first: %u) listed in `spell_ranks` already has ChainEntry from dbc.